RFC-000-rfc-process-suggestions.md

```diff
@@ -0,0 +1,77 @@
+# RFC Zero - Open for suggestions to the RFC process
+
+## Proposed by
+
+[Alyssa Rock](https://thegooddocs.slack.com/team/U012KCMPP0V)
+
+## Current status
+
+- [x] Draft
+- [x] Under discussion (until 2021-07-28)
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+
+## Proposal overview
+
+This RFC is open to suggestions from the community for minor and major suggestions to improve the RFC process.
+To make your suggestion to the process, add a comment to the "Feedback" section of this file or to this PR.
+
+
+## Motivation
+
+We've completed the RFC process with at least one RFC now and we've learned a few lessons along the way.
+Some members of the community expressed a desire to make a few minor tweaks and improvements and this RFC is designed to facilitate that discussion.
+
+
+## Proposal
+
+Make your suggestions here.
+Near the end of the community discussion period, I'll take the suggestions and add them to this section.
+After that, we'll open the final comment period and discuss the final changes.
+
+
+## Links
+
+No links necessary!
+
+## Feedback
+{{Reviewer name}}:
+- {{I suggest that ...}}
+    - {{Community member name}}: {{Additional comment on same topic.}}
+    - {{Proposer name}}: {{Addressed by ... / Ignored because ... / ...}}
+

```

Comment:
As I look at this section of the template, I find that our process is a little unclear about how to handle feedback.

- Is the proposer the person who makes the final call on whether to adopt or reject feedback? If so, do they resolve the conversation?
- If they adopt it, do they just incorporate it into their draft and provide attribution for the idea?
- If they reject it, do they note that in the feedback?
- At what stage do they do it they need to resolve the conversations and adopt/reject the feedback?

Reply:
yeah interesting points, I think it can be valuable to capture feedback that is meaningful and to re-incorporate back into the RFC especially if it belongs to another sections.  For instance, I mentioned above **Open Questions** those usually get added as I get comments on the RFC so I update that section with the questions being asked. And then work to update the main proposal sections as I answer those questions.

The __Feedback__ section -- I do think this could be helpful to have if there are conversations that we want to capture that didn't fit into any of the other sections.

Reply:
I've incorporated these comments into my RFC. Resolving.